annotation-file-utilities/lib/README

```diff
@@ -16,8 +16,11 @@ signature-util: https://repo.maven.apache.org/maven2/org/plumelib/signature-util
 checker-qual: https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/
   wget https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/2.5.4/checker-qual-2.5.4.jar
 
-# Use jsr308-langtools javac.jar rather than tools.jar, because some versions
-# of tools.jar contain bugs that have been fixed in new versions.  Also, we
-# have permission to distribute this jar.
-compiler: https://repo.maven.apache.org/maven2/org/checkerframework/compiler/
-  wget https://repo.maven.apache.org/maven2/org/checkerframework/compiler/2.4.0/compiler-2.4.0.jar
+javac-9-dev-r4023-3.jar: https://repo.maven.apache.org/maven2/com/google/errorprone/javac/
```

Comment:
Note that I added `javac-9-dev-r4023-3.jar`, `javac.jar`, and `javac-jdk.jar` to the repo and removed `compiler-2.4.0.jar`.